### ISSUE TITLE
[BE] When VVA service is down, display a message to users that manifests won't contain VVA documents

### ISSUE THREAD

**Issue by aroltsch** [department-of-veterans-affairs/caseflow-efolder#652]
Currently when VVA service is down, when the user tried to download the manifest, the download gets a status `vva_connection_error` and the user cannot proceed with downloading VBMS documents. The decision has been made to allow user to continue. We will just display a degradation banner that VVA is down. 
# AC
1) When VVA is down, the degradation banner is displayed. Design here: https://github.com/department-of-veterans-affairs/caseflow-efolder/issues/676

**Comment by aroltsch** [comment]
In order for us to turn on/off VVA globally, we will probably need to leave a FeatureToggle for `vva_service`.

**Comment by aroltsch** [comment]
relates #307

**Comment by aroltsch** [comment]
Waiting on design

**Comment by aroltsch** [comment]
Duplicate of #676
